slack/web/client.py

```diff
@@ -1367,16 +1367,17 @@ def mpim_replies(
         return self.api_call("mpim.replies", http_verb="GET", params=kwargs)
 
     def oauth_v2_access(
-        self, *, client_id: str, client_secret: str, code: str, **kwargs
+        self, *, client_id: str, client_secret: str, code: str, redirect_uri: str, **kwargs
     ) -> Union[Future, SlackResponse]:
         """Exchanges a temporary OAuth verifier code for an access token.
 
         Args:
             client_id (str): Issued when you created your application. e.g. '4b39e9-752c4'
             client_secret (str): Issued when you created your application. e.g. '33fea0113f5b1'
             code (str): The code param returned via the OAuth callback. e.g. 'ccdaa72ad'
+            redirect_uri (str): Must match the originally submitted URI (if one was sent). e.g. 'https://example.com'
         """
-        kwargs.update({"code": code})
+        kwargs.update({"code": code, "redirect_uri": redirect_uri})
```

Comment:
>A potential gotcha: while redirect_uri is optional, it is required if your app passed it as a parameter to oauth/authorization in the first step of the OAuth flow.

https://api.slack.com/methods/oauth.v2.access#arg_redirect_uri

As described in the above page, it's also possible to omit the `redirect_uri` parameter when starting OAuth flow. If this named arg is optional, I'm fine with this change for more developer-friendliness.